README.md

```diff
@@ -135,21 +153,26 @@ The [benchmark](https://github.com/Geal/pestvsnom) uses
 [a large 2MB JSON file](https://github.com/miloyip/nativejson-benchmark/blob/master/data/canada.json).
 Tested on a 2.6GHz Intel® Core™ i5 running macOS.
 
+The second entry is mapping Pairs, pest's internal lazy tree representation, to a custom JSON AST. While the first entry
+forms a perfectly usable tree, it does not produce a fully-processed JSON object, with decimal to binary number
+conversion notably missing. The second one does, but since it has an extra intermediate representation of the object,
+it repeats some work by performing extra allocations.
+
 ## Other features
 
-* precedence climbing
-* input handling
-* custom errors
-* runs on stable Rust
+* Precedence climbing
+* Input handling
+* Custom errors
+* Runs on stable Rust
 
 ## Usage
-pest requires [Cargo and Rust 1.23](https://www.rust-lang.org/en-US/downloads.html).
+pest requires [Cargo and Rust 1.23+](https://www.rust-lang.org/en-US/downloads.html).
 
 Add the following to `Cargo.toml`:
 
 ```toml
-pest = "^1.0"
-pest_derive = "^1.0"
+pest = "^2.0"
```

Comment:
We should probably remove caret dependency usage throughout the project since it seems to default to it anyway.

Reply:
Will remove 👍